### PR TITLE
Getting historical images when creating new field in the satellite

### PR DESCRIPTION
**Pull request by acebr1**

### CODE REVIEW

File: docs/satellite_endpoints.md [hunk 1]
```diff
@@ -281,15 +281,19 @@ curl -X GET \
...
+For deleting, use a `DELETE` HTTP request.
 
 :::caution
 
-Note that the field deletion cascades to all other tables. Meaning that all
-processed images will be lost.
+Note that the field deletion is irreversible and all images will be lost.
+
+But you can always create a new field and get images from the past, as far
+as you want.
```
Comment (acebr1) [inline docs/satellite_endpoints.md:296]: Nice

File: docs/satellite_endpoints.md [hunk 2]
```diff
@@ -310,9 +314,28 @@ The payload of this object should be like the following:
...
+
+By default, Leaf will retrieve and return images for your field from the last 
+30 days (from the moment you create the field). 
+
+This process will take place in the background.
+
 There are some limitations regarding the geometry of the field. It cannot be
-bigger than 50 million square meters and it cannot have a perimeter bigger
-than ~28 thousand meters.
+bigger than 5k hectares (12.3k acres) and it cannot have a perimeter bigger
```
Comment (meyer1994) [inline docs/satellite_endpoints.md:337]: 50k hectares

File: docs/satellite_endpoints.md [hunk 3]
```diff
@@ -310,9 +314,28 @@ The payload of this object should be like the following:
...
+than 28km (17.4 miles).
```
Comment (meyer1994) [inline docs/satellite_endpoints.md:338]: 89km, which is the result of:
```py
sqrt(500_000_000) * 4
```
Which is the perimeter of an square with 50k hecates of area.

File: docs/satellite_overview.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ Leaf's service uses Sentinel-2 data which has the highest resolution publicly av
 <img alt="Field example" src={useBaseUrl('img/fieldovertime.png')} />
 
 This API accepts field boundary polygons and returns processed, cropped, and color-corrected RGB and NDVI images every time there is a new image available.
-We also provide information like percent cloud coverage of the field and the Sentinel tile source for each image.
+We also provide information like percent cloud coverage of the field, and the Sentinel tile source for each image.
```
Comment (meyer1994) [inline docs/satellite_overview.md:17]: Data coverage percentage as well.

**Review by meyer1994 — CHANGES_REQUESTED** [review]

File: docs/satellite_endpoints.md (hunk 2 above)
Comment (acebr1) [reply docs/satellite_endpoints.md:337]: Captura de tela de 2020-09-10 14-01-25

Comment (acebr1) [reply docs/satellite_endpoints.md:337]: Captura de tela de 2020-09-10 14-02-47

File: docs/satellite_endpoints.md (hunk 3 above)
Comment (acebr1) [reply docs/satellite_endpoints.md:338]: sqrt(50_000_000) * 4 = 28284

**Review by meyer1994 — APPROVED** [review]

**Review by GMBueno — APPROVED** [review]

**Pull request merged**